Last-contracts/scripts/hatchEggs.js

```diff
@@ -12,12 +12,12 @@ function sleep(milliseconds) {
 module.exports = async () => {
   try {
     let eggContract = await Egg.deployed()
-    let eggOne = await eggContract.hatch({from: web3.eth.accounts[1]})
-    let eggTwo = await eggContract.hatch({from: web3.eth.accounts[1]})
-    let eggThree = await eggContract.hatch({from: web3.eth.accounts[1]})
+    let eggOne = await eggContract.hatch(1, {from: web3.eth.accounts[1]})
```

Comment:
This is fine for now krub. But probably in the future- we might want to have script command take arguments for amount to hatch.

ie, `node hatchEggs.js 100` to mint 100 eggs